e2e/vpc/vpc_create_volume_custom_profile.go

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
+ * IBM Confidential
+ * OCO Source Materials
+ * IBM Cloud Container Service, 5737-D43
+ * (C) Copyright IBM Corp. 2018 All Rights Reserved.
+ * The source code for this program is not  published or otherwise divested of
+ * its trade secrets, irrespective of what has been deposited with
+ * the U.S. Copyright Office.
+ ******************************************************************************/
+
+package vpc
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"go.uber.org/zap"
+
+	"github.com/IBM/ibmcloud-storage-volume-lib/lib/provider"
+)
+
+var _ = Describe("ibmcloud-storage-volume-lib", func() {
+	It("VPC: Create and delete VPC volume[profile: custom]", func() {
+		volName := volumeName + "custom"
+		volSize := volumeSize
+		Iops := iops
+
+		volume := &provider.Volume{}
+
+		volume.VolumeType = volumeType
+		volume.VPCVolume.Generation = generation
+		volume.VPCVolume.ResourceGroup = &provider.ResourceGroup{}
+		volume.VPCVolume.Profile = &provider.Profile{Name: "custom"}
+		volume.Name = &volName
+		volume.Capacity = &volSize
```

Comment:
Can we verify custom class with 2-3 different volume size as well

Reply:
Sure